indexer/bin/indexer.js

```diff
@@ -79,8 +79,17 @@ export default {
         return new Response('Bad Request', { status: 400 })
       }
 
+      if (!Array.isArray(payload.metadata_keys)) {
+        console.error(
+          `Unexpected payload.metadata_keys value ${JSON.stringify(payload.metadata_keys)}`,
+        )
+        return new Response('Bad Request: Invalid metadata_keys', {
+          status: 400,
+        })
+      }
```

Comment:
Inconsistent: We have a special handler just for `metadata_keys` but a generic one for the other fields. Could we have either or?